calico_containers/tests/unit/datastore_test.py

```diff
@@ -9,7 +9,9 @@
                                                  Endpoint,
                                                  NoEndpointForContainer,
                                                  CALICO_V_PATH,
-                                                 DataStoreError)
+                                                 DataStoreError,
+                                                 BGP_NODE_MESH_PATH,
```

Comment:
I'm not very keen on importing path constants from the module under test.  The UTs should have their own constants that come directly from the spec.